src/ros_filter.cpp

```diff
@@ -2002,14 +2002,28 @@ void RosFilter<T>::initialize()
     tf2::toMsg(tf2::Transform::getIdentity());
 
   // Position publisher
+  rclcpp::PublisherOptions publisher_options;
+  publisher_options.qos_overriding_options = rclcpp::QosOverridingOptions {{
```

Comment:
There's too much redundancy of this list. Can this not be included in a utils file that all of the uses of the list can be grabbed from? I think the "right" answer is to have the `get_default_overridable_options()` (or something similar) function in rclcpp itself, this seems like something **a ton** of people will need / want.

Reply:
Good call. I'll make a PR to `rclcpp` with something similar.

Reply:
There is the method [`with_default_policies`](https://github.com/ros2/rclcpp/blob/0251f70fe8a952fae242e6dcdc5b11bc4c11dc76/rclcpp/include/rclcpp/qos_overriding_options.hpp#L141) that I suggest we call here instead (unless someone feels strongly about exposing configuration points for QoS settings other than history, depth, and reliability).